src/reference/modules/administration/pages/multi_server.adoc

```diff
@@ -1,188 +1,110 @@
+== On the root server
 
-----
+=== Initialize the database
 
-* Restart postgresql to apply the new settings:
+The provisioning script is located on the root server in `/opt/rudder/share/upgrade-tools/reportsSchema.sql`.
 
-----
-
-systemctl restart postgresql
+If you database has a name different from `rudder`, you need to make a copy of this file
+and edit it. Change the line:
 
 ----
 
-* Execute the following command to configure the password (that should be the same as RUDDER_PSQL_PASSWORD in `/opt/rudder/etc/rudder-passwords.conf` on the root server):
+ALTER database rudder SET standard_conforming_strings=true;
 
 ----
 
-su - postgres -c "psql -c \"ALTER USER rudder WITH PASSWORD 'RUDDER_SERVER_DATABASE_PASSWORD'\""
+To:
 
 ----
 
-* Run an inventory to the server:
+ALTER database alternate_base_name SET standard_conforming_strings=true;
 
 ----
 
-rudder agent inventory
+Then initialize the database:
 
 ----
 
-=== On the root server
-
-In the following section, DATABASE_HOST refers to the hostname of the new database server, and SERVER_HOST to the hostname of
-the root server.
-
-* Remove the rudder-server-root and rudder-reports packages if installed. For example, you can run on Debian:
+psql -q -U DB_USER -h DB_HOST -d DB_NAME -f reportsSchema.sql
 
 ----
 
-systemctl restart rudder-server
-apt-mark manual rudder-webapp
-apt-get remove --purge rudder-reports
+* Execute the following command to configure the password (that should be the same as RUDDER_PSQL_PASSWORD in `/opt/rudder/etc/rudder-passwords.conf` on the root server):
 
-----
+=== Configure the database
 
-* You can also remove the postgresql package and database from the server if installed, but keep in mind you will lose all existing data.
-You can follow the xref:administration:procedures.adoc#_migration_backups_and_restores[backup and restore] procedure to migrate the data to the new database.
+* Set the database password in `/opt/rudder/etc/rudder-passwords.conf, on the `RUDDER_PSQL_PASSWORD` line.
 
-* Change the hostname in `/opt/rudder/etc/rudder-web.properties`:
+* Run the agent on the root server to synchronize the password change:
 
 ----
 
-rudder.jdbc.url=jdbc:postgresql://DATABASE_HOST:5432/rudder
+rudder agent run
 
 ----
 
-* Edit `/var/rudder/cfengine-community/inputs/rudder-server-roles.conf` and set the following line:
+* Change the hostname in `/opt/rudder/etc/rudder-web.properties`, and database name and user if needed:
 
 ----
 
-rudder-db:DATABASE_HOST
+# Don't try to configure a local postgresql on the root server
+rudder.postgresql.local=false
 
-----
+# Set connection details
+rudder.jdbc.url=jdbc:postgresql://DB_HOST:5432/DB_NAME
+rudder.jdbc.username=DB_USER
 
-* Run an inventory:
+# This one is synchronized by the agent from rudder-passwords.conf
+rudder.jdbc.password=DB_PASSWORD
 
 ----
 
-rudder agent inventory
+* Restart the server service:
 
 ----
 
-* Restart rudder services:
+systemctl restart rudder-jetty
 
 ----
 
```

Comment:
I would had something about how to check that it worked: "Rudder check DB connection when starting up, so if something is misconfigured, you will see an error message in /var/log/rudder/webapp/xxx and jetty will stop"

Reply:
:+1: